mne/io/ctf/tests/test_ctf.py

```diff
@@ -12,13 +12,15 @@
 from numpy.testing import assert_allclose, assert_array_equal
 import pytest
 
-from mne import pick_types
+from mne import (pick_types, read_annotations, create_info,
+                 events_from_annotations)
 from mne.transforms import apply_trans
-from mne.io import read_raw_fif, read_raw_ctf
+from mne.io import read_raw_fif, read_raw_ctf, RawArray
 from mne.io.compensator import get_current_comp
 from mne.io.tests.test_raw import _test_raw_reader
+from mne.tests.test_annotations import _assert_annotations_equal
```

Comment:
we have `_assert_annotations_equal` defined more than once. I think that we could define our own assertion like [this with pytest](http://doc.pytest.org/en/latest/assert.html#defining-your-own-explanation-for-failed-assertions).
Then we would be able to write:
```py
assert raw.annotations == annot
```

We cannot do it with the MNE objects that inherit from dict. But annotations does not.
I'm not planning to do this here. This should be done in a different PR.

Reply:
opinions?